content/collections.md

```diff
@@ -435,6 +435,53 @@ As we discussed earlier, it's very common in Meteor applications to have associa
 
 To make this easier, we can attach functions to the prototype of the documents that belong to a given collection, to give us "methods" on the documents (in the object oriented sense). We can then use these methods to create new queries to find related documents.
 
+To make things even easier, we can use the [`cultofcoders:grapher`](https://atmospherejs.com/cultofcoders/grapher) package to easily associate collections and easily fetch their relations for example:
+
+```js
+// Configure how collections relate to each other
+Todos.addLinks({
+    list: {
```

Comment:
Please adjust code indentation to use 2 spaces, to line up with the rest of the file.